.codespellrc

```diff
@@ -1,3 +1,3 @@
 [codespell]
 skip = ./testing/**/*,./.git/**/*,./application/cxxopts.hpp,./cmake/NSIS.template.in
-ignore-words-list=nnumber,unknwn,dota
+ignore-words-list=nnumber,unknwn,dota,superseed
```

Comment:
Why do you want to whitelist it instead of fixing errors?

Reply:
superseed is a word

Reply:
It's not ^^

Reply:
turns out its not